test/jdk/java/nio/MappedByteBuffer/PmemTest.java

```diff
@@ -77,23 +77,18 @@
  *  make test TEST=jdk/java/nio/MappedByteBuffer/PmemTest.java
 */
 
-/* @test
+/* @test id=default_architecture_test_case
  * @summary Testing NVRAM mapped byte buffer support
  * @run main/manual PmemTest
  * @requires (os.family == "linux")
- * @requires (os.arch == "x86_64")
+ * @requires ((os.arch == "x86_64")|(os.arch == "amd64"))
  */
 
-/* @test
+/* @test id=other_architectures_test_case
```

Comment:
we should to leave the @ignores for the 2nd test case 
the test will be run on amd64 environmenst , as such the addition of  | (os.arch == "amd64") is fine

Reply:
so just to confirm: we explitcitly want to ignore this test on `linux-aarch64` and `linux-ppc64le` regardless of is the required hardware present or not?

Reply:
yes, retain the @ignore in the second test case.
The original aim of this issue is to change the default test case of os.arch == "x86_64" to os.arch == "amd64" as the test will by default be run in an amd64 environment.